layouts/shortcodes/official-cve-feed.html

```diff
@@ -0,0 +1,19 @@
+<table class="security-cves">
+  <caption>{{ T "cve_table" }}</caption>
+  <thead>
+  <tr>
+    <th>{{ T "cve_id" }}</th>
+    <th>{{ T "cve_summary"}}</th>
+    <th>{{ T "cve_issue_url" }}</th>
+  </tr>
+  </thead>
+  <tbody>
+  {{ range $issues := getJSON "https://storage.googleapis.com/fake-test-cve-feed-kep-3203/official-cve-feed.json" }}
```

Comment:
Ideally, we'd make this a site parameter (see the top-level `config.toml`) so that we can change / configure it in one place.

Reply:
Added. It does look cleaner